Source/Scene/GlobeSurfaceTile.js

```diff
@@ -337,14 +338,16 @@ define([
             isDoneLoading = isDoneLoading && thisTileDoneLoading;
 
             // The imagery is renderable as soon as we have any renderable imagery for this region.
-            isRenderable = isRenderable && (thisTileDoneLoading || defined(tileImagery.readyImagery));
+            isAnyTileLoaded = isAnyTileLoaded || (thisTileDoneLoading || defined(tileImagery.readyImagery));
 
             isUpsampledOnly = isUpsampledOnly && defined(tileImagery.loadingImagery) &&
                               (tileImagery.loadingImagery.state === ImageryState.FAILED || tileImagery.loadingImagery.state === ImageryState.INVALID);
         }
 
         tile.upsampledFromParent = isUpsampledOnly;
-        tile.renderable = isRenderable;
+
+        // Allow rendering if any available layers are loaded
+        tile.renderable = isRenderable && isAnyTileLoaded;
```

Comment:
No more need for `isRenderable` variable; this reads more cleanly as `tile.renderable &= isAnyTileLoaded` (or `tile.renderable = tile.renderable && isAnyTileLoaded`).

Reply:
That's true. I would lean toward the latter option to maintain the boolean.